nncf/quantization/algorithms/min_max/algorithm.py

```diff
@@ -69,6 +69,7 @@ class MinMaxQuantizationParameters(AlgorithmParameters):
                                       per_channel=False)
 
     def __init__(self,
+                 inplace_statistics: bool,
```

Comment:
Does this parameter have a default value or user should always specify it?

Reply:
The idea is following: Default way to specify PTQ parameters is `PostTrainingQuantizationParameters`. When user specifies `PostTrainingQuantizationParameters`, `inplace_statistics=False` is applied by default as this feature not implemented in all backends. `MinMaxParameters` and `BiasCorrectionParameters` are parameters for advanced users (`test_quantize_conformance`, for example), and here it's better to specify this parameter explicitly

Reply:
Changed it